spk/wallabag/Makefile

```diff
@@ -0,0 +1,45 @@
+SPK_NAME = wallabag
+SPK_VERS = 1.9.1-b
```

Comment:
We need to be careful with letters and special characters in version number, the version comparison in DSM is somewhat tricky and with this 1.9.2 could be considered a lower version than 1.9.1-b.
I had some problems with that in the past